myproject/blog/models.py

```diff
@@ -1,24 +1,26 @@
 from django.db import models
 from django.utils import timezone
+from django.contrib.auth.models import User
 
 class BaseModel(models.Model):
-    created_at = models.DateTimeField(auto_now_add=True)
+    created_at = models.DateTimeField(default=timezone.now, editable=False)
     updated_at = models.DateTimeField(auto_now=True)
 
     class Meta:
         abstract = True
         db_table = "base_model"
 
 class Author(BaseModel):
+    user = models.OneToOneField(User, on_delete=models.CASCADE, default=None)
     name = models.CharField(max_length=50)
     biological_info = models.TextField()
-    
+
     class Meta:
         db_table = "author"
-        
+
     def __str__(self):
         return self.name
-    
+
 class BlogPost(BaseModel):
     blog_title = models.CharField(max_length=200)
     post_date = models.DateTimeField("date posted")
```

Comment:
You don't need this column, as this can be handled by `created_at` column of base model.